specs/Print.md

```diff
@@ -0,0 +1,425 @@
+  /// job is in progress, the completed handler is immediately invoked with an error.
+  ///
+  /// \snippet AppWindow.cpp PrintToPdfStream
+  HRESULT PrintToPdfStream([in] ICoreWebView2PrintSettings2* printSettings,
+                           [in] ICoreWebView2PrintToPdfStreamCompletedHandler* handler);
+}
+
+/// Receives the result of the `PrintWithSettings` method.
+/// `errorCode` returns S_OK if the PrintWithSettings operation succeeded.
+[uuid(4B2FE043-1A1A-4924-803D-C05E5FDCA3B0), object, pointer_default(unique)]
+interface ICoreWebView2PrintWithSettingsCompletedHandler : IUnknown {
+  /// Provides the result of the corresponding asynchronous method. 
+  HRESULT Invoke([in] HRESULT errorCode);
+}
+
+/// Receives the result of the `PrintToPdfStream` method.
+/// `errorCode` returns S_OK if the PrintToPdfStream operation succeeded.
+/// The printable pdf data is returned in the `pdfStream` object.
+[uuid(78A9FC37-249B-4E7E-A2E6-20CAAA14AE7B), object, pointer_default(unique)]
+interface ICoreWebView2PrintToPdfStreamCompletedHandler : IUnknown {
+  /// Provides the result of the corresponding asynchronous method.
+  HRESULT Invoke([in] HRESULT errorCode, [in] IStream* pdfStream);
+}
+
+/// Settings used by the `PrintWithSettings` method.
+[uuid(DA3AD25C-D121-44DC-83DA-BF5E50DBF0DF), object, pointer_default(unique)]
+interface ICoreWebView2PrintSettings2 : ICoreWebView2PrintSettings {
+  /// Page range to print, e.g., `1-5, 8, 11-13`. Defaults to empty string,
+  /// which means print all pages. Invalid pages are ignored.
+  /// 
+  /// The caller must free the returned string with `CoTaskMemFree`. See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings)
+  [propget] HRESULT PageRanges([out, retval] LPWSTR* value);
+
+  /// Set the `PageRanges` property.
+  [propput] HRESULT PageRanges([in] LPCWSTR value);
+
+  /// Number of pages per sheet. The Default value is 1 and maximum is 16.
+  [propget] HRESULT PagesPerSheet([out, retval] INT32* value);
+
+  /// Set the `PagesPerSheet` property. Returns `E_INVALIDARG` if an invalid value is
+  /// provided, and the current value is not changed.
+  [propput] HRESULT PagesPerSheet([in] INT32 value);
+
+  /// Number of copies to print. The Default value is 1 and maximum 
+  /// copies count is 999. 
+  /// This value is ignored in PrintToPdfStream method.
+  [propget] HRESULT Copies([out, retval] INT32* value);
+
+  /// Set the `Copies` property. Returns `E_INVALIDARG` if an invalid value is provided 
+  /// for the specified printer, and the current value is not changed.
+  [propput] HRESULT Copies([in] INT32 value);
+
+  /// Collate of the printer. The default value is `FALSE`. 
+  /// This value is ignored in PrintToPdfStream method.
+  [propget] HRESULT Collate([out, retval] BOOL* value);
+
+  /// Set the `Collate` property.
+  [propput] HRESULT Collate([in] BOOL value);
+
+  /// Color of the print. The default value is `TRUE`.
```

Comment:
This text makes it sound like the property is picking a color. But it seems like its actually picking if the print should be in color or black and white?